pkgs/top-level/python-packages.nix

```diff
@@ -2770,6 +2770,9 @@ in {
 
   future = callPackage ../development/python-modules/future { };
 
+  # Defined only for Python 2 in ./python2-packages.nix
+  futures = null;
```

Comment:
I don't think this is a good idea. We should at least use a throw.

Reply:
Agreed, will change this.